CoreAutomata/AutomataApi.cs

```diff
@@ -180,6 +192,11 @@ public static void ShowMessageBox(string Title, string Message)
             _platformImpl.MessageBox(Title, Message);
         }
 
+        public static void Highlight(Region Region, double Timeout)
+        {
+            _platformImpl.Highlight(Region.Transform(), Timeout);
```

Comment:
All the other ones use Region.Highlight() but this one uses _platformImpl.Highlight() with a transformed region?